app/src/main/res/values/strings.xml

```diff
@@ -486,4 +498,5 @@
     <string name="server_setup_incomplete">The setup of this server has not been completed. Open Jellyfin in a web browser to finish setup before signing in.</string>
     <string name="enable_picture_viewer_title">Enable new picture viewer</string>
     <string name="episode_name_special">special</string>
+    <string name="lbl_subtitle_position">Subtitle Position from Bottom</string>
```

Comment:
Duplicate from line 236

Reply:
Now fixed must have got in there from there from when I moved the variables around so they are more contiguous